pkg/apiserver/handlers.go

```diff
@@ -75,7 +75,8 @@ func RateLimit(rl util.RateLimiter, handler http.Handler) http.Handler {
 		// Return a 429 status indicating "Too Many Requests"
 		w.Header().Set("Retry-After", "1")
 		w.WriteHeader(errors.StatusTooManyRequests)
-		fmt.Fprintf(w, "Rate limit is 1 QPS or a burst of 20")
+		// TODO(satnam6502): Replace this with plumbed-in flag values *apiRate and *apiBurst
```

Comment:
We won't have access to the flags here.  Instead, we should ask the rate limiter to describe itself and post that into the response, perhaps?